bitnami/ejbca/templates/deployment.yaml

```diff
@@ -0,0 +1,173 @@
+            - name: EJBCA_DATABASE_HOST
+              value: {{ template "ejbca.databaseHost" . }}
+            - name: EJBCA_DATABASE_PORT
+              value: {{ template "ejbca.databasePort" . }}
+            - name: EJBCA_DATABASE_NAME
+              value: {{ template "ejbca.databaseName" . }}
+            - name: EJBCA_DATABASE_USERNAME
+              value: {{ template "ejbca.databaseUsername" . }}
+            {{- if .Values.ejbcaKeystoreExistingSecret }}
+            - name: EJBCA_SERVER_CERT_FILE
+              value: /opt/bitnami/ejbca/mounted-jks/keystore.jks
+            {{- end }}
+            {{- if .Values.ejbcaJavaOpts }}
+            - name: JAVA_OPTS
+              value: {{ .Values.ejbcaJavaOpts | quote }}
+            {{- end }}
+            {{- if .Values.ejbcaCA }}
+            {{- if .Values.ejbcaCA.name }}
+            - name: EJBCA_CA_NAME
+              value: {{ .Values.ejbcaCA.name | quote }}
+            {{- end }}
+            {{- if .Values.ejbcaCA.baseDN }}
+            - name: EJBCA_BASE_DN
+              value: {{ .Values.ejbcaCA.baseDN | quote }}
+            {{- end }}
+            {{- end }}
+            {{- if .Values.extraEnv }}
+            {{- include "ejbca.tplValue" (dict "value" .Values.extraEnv "context" $) | nindent 12 }}
+            {{- end }}
+          ports:
+            - name: http
+              containerPort: 8080
+            - name: https
+              containerPort: 8443
+          {{- if .Values.livenessProbe.enabled }}
+          livenessProbe:
+            initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
+            successThreshold: {{ .Values.livenessProbe.successThreshold }}
+            failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
+            httpGet:
+              path: /ejbca
+              port: http
+          {{- else if .Values.customLivenessProbe }}
+          livenessProbe: {{- include "ejbca.tplValue" (dict "value" .Values.customLivenessProbe "context" $) | nindent 12 }}
+          {{- end }}
+          {{- if .Values.readinessProbe.enabled }}
+          readinessProbe:
+            initialDelaySeconds: {{ .Values.readinessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.readinessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.readinessProbe.timeoutSeconds }}
+            successThreshold: {{ .Values.readinessProbe.successThreshold }}
+            failureThreshold: {{ .Values.readinessProbe.failureThreshold }}
+            httpGet:
+              path: /ejbca
+              port: http
+          {{- else if .Values.customReadinessProbe }}
+          readinessProbe: {{- include "ejbca.tplValue" (dict "value" .Values.customReadinessProbe "context" $) | nindent 12 }}
+          {{- end }}
+          volumeMounts:
```

Comment:
Would it make sense to support custom init scripts (raw in values.yaml with `initScripts`, secrets with `initScriptsSecret` and configmaps with `initScriptsCM`)?

Reply:
It would be a nice idea for sure, but AFAIK the main container does not support initScripts.